packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -551,7 +551,7 @@ const CFDPasswordForm = (props: TCFDPasswordFormProps) => {
                             />
                         </div>
 
-                        {props.is_real_financial_stp  && (
+                        {props.is_real_financial_stp && (
```

Comment:
can we de-structure the props here ?

Reply:
Rupato it is not my changes .... this is what eslint did... it happens if we don;t format out codes with orettier before pushing and then the others get the untouched files as changes.